### PR TITLE
fix bug 1224103

### PR DESCRIPTION
**Pull request by chikoski**
add ja-locale specific rule to media/css/firefox/ios.less

### CODE REVIEW

**Comment by kyoshino** [comment]
The layout looks good to me.

File: media/css/firefox/ios.less [hunk 1]
```diff
@@ -840,3 +840,10 @@ a.go {
         .background-size(210px 290px);
     }
 }
+
+// Bug 1224103
+html[lang=ja]{
+  #privacy ul p{
```
Comment (kyoshino) [inline media/css/firefox/ios.less]: Please add a space before those curly brackets `{` for consistency.

File: media/css/firefox/ios.less [hunk 2]
```diff
@@ -840,3 +840,10 @@ a.go {
...
+    text-align: left;
+  }
+}
```
Comment (kyoshino) [inline media/css/firefox/ios.less]: Add a line break here.

File: media/css/firefox/ios.less (hunk 1 above)
Comment (chikoski) [inline media/css/firefox/ios.less]: Thanks!

**Comment by kyoshino** [comment]
Hm, one more thing... Looks like the indent in this file is 4 spaces. Can you fix it as well?

**Comment by chikoski** [comment]
@kyoshino Thanks for reviewing my code.
I changed the tab size in my update.

**Comment by kyoshino** [comment]
Thanks @chikoski :+1:

**Comment by craigcook** [comment]
I amended the commit message to add a bit more context and merged this change to master in https://github.com/mozilla/bedrock/commit/25cd6443f6138c63103280a072de7246c7410e70 
Thanks!

**Pull request closed without merging**